web/src/routes/news/containers/TunewsDetailsPage.js

```diff
@@ -82,8 +81,15 @@ export const TunewsDetailsPage = ({ tunewsElement, language, id, city, cities }:
             <StyledBannerImage src={TunewsIcon} alt='' />
           </StyledTitle>
         </StyledBanner>
-        <Title>{title}</Title>
-        <Content>{content}</Content>
+        <Page
+          title={title}
+          content={filteredContent}
+          formatter={formatter}
+          lastUpdateFormat={lastUpdateDateFormat}
+          lastUpdate={date}
+          showLastUpdateText={false}
+          onInternalLinkClick={push}
+        />
       </StyledWrapper>
       <TunewsDetailsFooter eNewsNo={eNewsNo} date={date} formatter={formatter} />
```

Comment:
JFYI: The custom footer will be removed in a separate issue, just in case you were wondering why there is duplicated information here.